lib/carthage_cache/configuration.rb

```diff
@@ -25,7 +29,8 @@ def self.default
         aws_s3_client_options: {
           region: ENV['AWS_REGION'],
           access_key_id: ENV['AWS_ACCESS_KEY_ID'],
-          secret_access_key: ENV['AWS_SECRET_ACCESS_KEY']
+          secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
+          profile: ENV['AWS_PROFILE']
```

Comment:
It's if you handle AWS credentials using the credentials file in `~/.aws/credentials` instead of having them as env variables. It's convenient if you have different accounts or environments.

So if you have in your home a credentials file with content

```
[personal]
aws_access_key_id = YOUR_ACCESS_KEY_ID
aws_secret_access_key = YOUR_SECRET_ACCESS_KEY
[work]
aws_access_key_id = YOUR_ACCESS_KEY_ID
aws_secret_access_key = YOUR_SECRET_ACCESS_KEY
```

You can easily switch in your console between them by specifying either `AWS_PROFILE=personal` or `AWS_PROFILE=work`

[More info](https://blogs.aws.amazon.com/security/post/Tx3D6U6WSFGOK2H/A-New-and-Standardized-Way-to-Manage-Credentials-in-the-AWS-SDKs)